legend-engine-xts-relationalStore/legend-engine-xt-relationalStore-dbExtension/legend-engine-xt-relationalStore-bigquery/legend-engine-xt-relationalStore-bigquery-wrapper/src/test/resources/sql_test_summary.md

```diff
@@ -0,0 +1,229 @@
+Legend SQL Compatibility Report
```

Comment:
This test summary md file will become stale as the codebase evolves.

Please remove from the PR. Might be good to attach to the PR as a doc attachment.